### PR TITLE
[CVP-2488]-adding verbose option to operator-sdk command

### PR DESCRIPTION
**Pull request by jsztuka**
Adding verbose output to operator-sdk scorecard command due to debugging purposes.

### CODE REVIEW

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh [hunk 1]
```diff
@@ -33,7 +33,8 @@ operator-sdk scorecard --config "${SCORECARD_CONFIG}" \
                        --namespace "${NAMESPACE}" \
                        --kubeconfig "${KUBECONFIG}" \
                        --output json \
-                       "${OPERATOR_DIR}" > "${ARTIFACT_DIR}"/scorecard-output-basic.json || true
+                       "${OPERATOR_DIR}" > "${ARTIFACT_DIR}"/scorecard-output-basic.json || true \
+                       --verbose
```
Comment (lslebodn) [inline ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh:37]: @jsztuka you need to pass `--verbose` to the `operator-sdk` and not to the `/bin/true`

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh [hunk 2]
```diff
@@ -58,5 +59,6 @@ if [ -f "${OPERATOR_DIR}/tests/scorecard/config.yaml" ]; then
     --output json \
     --wait-time 3000s \
     --service-account "${SCORECARD_SERVICE_ACCOUNT}" \
-    "${OPERATOR_DIR}" > "${ARTIFACT_DIR}"/scorecard-output-custom.json || true
+    "${OPERATOR_DIR}" > "${ARTIFACT_DIR}"/scorecard-output-custom.json || true \
+    --verbose
```
Comment (lslebodn) [inline ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh:63]: @jsztuka you need to pass `--verbose` to the `operator-sdk` and not to the `/bin/true`

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh (hunk 1 above)
Comment (jsztuka) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh]: corrected

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh (hunk 2 above)
Comment (jsztuka) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh]: modified

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh [hunk 3]
```diff
@@ -32,8 +32,10 @@ echo "Running the operator-sdk scorecard test using the basic configuration, jso
 operator-sdk scorecard --config "${SCORECARD_CONFIG}" \
                        --namespace "${NAMESPACE}" \
                        --kubeconfig "${KUBECONFIG}" \
+                       --verbose \
                        --output json \
                        "${OPERATOR_DIR}" > "${ARTIFACT_DIR}"/scorecard-output-basic.json || true
+
```
Comment (lslebodn) [inline ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh:38]: nitpick; unnecessary extra blank line

**Comment by jsztuka** [comment]
@sonam1412

**Review by dirgim — APPROVED** [review]

**Pull request merged**